hw/ip/rv_timer/data/rv_timer_testplan.hjson

```diff
@@ -67,5 +67,36 @@
       stage: V2
       tests: ["rv_timer_stress_all"]
     }
+    {
+      name: tick _configuration
+      desc: '''Verify that the timer can be configured to generate a system tick.
+      
+            - Configure to generate a tick of `T`.
+            - Enable the timer.
+            - Start a busy loop of 1 second based on the `mcycleh` csr.
+            - Read the number of ticks `n`.
+            - Verify that `n * T ~= 1 second` with 5% of tolerance.
+            - Reapeat for T in :
+                [1us, 10us, 100us and 1ms]
+            '''
+      stage: V3
```

Comment:
You should also mention a `si_val` stage, probably SV2.

Reply:
Done